### PR TITLE
Add dumpling dipping sauce

### PR DESCRIPTION
**Pull request by huln24**

### CODE REVIEW

File: docs/sauces/dipping.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Dipping Sauces
+==============
+
+Dumpling
```
Comment (Delaunay) [inline docs/sauces/dipping.rst:4]: Nice

Comment (huln24) [reply docs/sauces/dipping.rst:4]: ok done

**Comment by Delaunay** [comment]
Can you add the page to the sauce index ?

**Comment by huln24** [comment]
> 

Done!

**Pull request merged**